subprojects/gst-plugins-bad/sys/msdk/gstmsdkenc.c

```diff
@@ -1615,13 +1606,12 @@ gst_msdkenc_set_format (GstVideoEncoder * encoder, GstVideoCodecState * state)
   return TRUE;
 }
 
-static MsdkSurface *
+static GstMsdkSurface *
 gst_msdkenc_get_surface_from_pool (GstMsdkEnc * thiz, GstBufferPool * pool,
     GstBufferPoolAcquireParams * params)
 {
   GstBuffer *new_buffer;
-  mfxFrameSurface1 *new_surface;
-  MsdkSurface *msdk_surface;
+  GstMsdkSurface *msdk_surface;
```

Comment:
on Windows it will remain uninitialized

Reply:
> on Windows it will remain uninitialized

Ok, I will initialize msdk_surface as NULL.